app/ts/SmaaTab/SmaaResults/RankAcceptabilities/RankAcceptabilities.tsx

```diff
@@ -13,8 +14,10 @@ export default function RankAcceptabilities(): JSX.Element {
           Rank acceptabilities <InlineHelp helpId="rank-acceptabilities" />
         </Typography>
       </Grid>
-      <Grid item xs={6}>
-        <RankAcceptabilitiesPlot />
+      <Grid container item xs={6}>
```

Comment:
container item, funny

Reply:
there are lots of these... :) anything funny in particular about this one?